README.md

```diff
@@ -256,6 +256,72 @@ to return forces.
 torch_force.setOutputsForces(True)
 ```
 
+Computing energy derivatives with respect to global parameters
+--------------------------------------------------------------
+
+Its possible to query `TorchForce` for the derivative of the energy with respect to global parameters. In order to do so the global parameters must be registered as energy derivatives. This is done by calling `addEnergyParameterDerivative()` for each parameter.
+
+The parameter derivatives can be queried by calling `getEnergyParameterDerivatives()` on the `State` object returned by `Context.getState()`. The result is a dictionary with the parameter names as keys and the derivatives as values.
+
+```python
+import torch as pt
+from torch import Tensor
+from openmmtorch import TorchForce
+import openmm as mm
+
+class ForceWithParameters(pt.nn.Module):
+
+    def __init__(self):
+        super(ForceWithParameters, self).__init__()
+
+    def forward(
+        self, positions: Tensor, parameter1: Tensor, parameter2: Tensor
+    ) -> Tensor:
+        x2 = positions.pow(2).sum(dim=1)
+        u_harmonic = ((parameter1 + parameter2**2) * x2).sum()
+        return u_harmonic
+
+
+def example():
```

Comment:
All the other examples are much more minimal than this.  They don't give complete test cases with assertions, just the essential code for TorchForce and explanations of what it does.  In this case we need to explain how to create the force and register the parameter derivative:

```python
module = torch.jit.script(ForceModule())
force = TorchForce(module)
force.addGlobalParameter("k", 2.0)
force.addEnergyParameterDerivative("k")
```

and then how to query it from the Context:

```python
state = context.getState(getParameterDerivatives=True)
dEdk = state.getEnergyParameterDerivatives()["k"]
```